### ISSUE TITLE
Failure in github action: This git repository has untracked files or uncommitted changes

### ISSUE THREAD

**Issue by lindskogen** [packtracker/report#5]
```
This git repository has untracked files or uncommitted changes:

frontend/package.json
M frontend/yarn.lock

Remove untracked files, stash or commit any changes, and try again.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
##[error]Docker run failed with exit code 1
```

Reproducible example: https://github.com/cthdidIT/filmstund/pull/385/checks?check_run_id=275106447

**Comment by jondavidjohn** [comment]
Hey @lindskogen sorry for the delay, I seem to have not set up notifications for this repo 🤦‍♂

Are you still fighting this issue? or did you find a solution?

**Comment by lindskogen** [comment]
No, the build is still failing, in the meantime you might want to do something about this: https://github.com/packtracker/report/pull/2

**Comment by jondavidjohn** [comment]
@lindskogen This should be fixed now with version 2.2.8

Thanks for raising this 👍
